docs/volume/edit-volume.md

```diff
@@ -20,4 +20,11 @@ To prevent the expansion from interference by unexpected data R/W, Harvester sup
 
 If you specify a size larger than Longhorn's capacity during the expansion, the status of the volume expansion will be stuck in `Resizing`. You can cancel the failed volume expansion by clicking the `⋮` button and selecting the `Cancel Expand` option.
 
-![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
\ No newline at end of file
+![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
+
+## Change the Storage Class of an Existing Volume
```

Comment:
s\Storage Class\StorageClass
can we change this word in all places to align with the k8s' official doc? thanks.

Reply:
Renamed. Screenshots in the docs and the UI also need to be updated.